bears/go/GoVetBear.py

```diff
@@ -25,5 +28,96 @@ class GoVetBear:
     CAN_DETECT = {'Unused Code', 'Smell', 'Unreachable Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return 'vet', filename
+    def create_arguments(filename, file, config_file,
+                         assembly_go_mismatch: bool=False,
+                         useless_assignment: bool=False,
+                         atomic: bool=False,
+                         boolean: bool=False,
```

Comment:
And that's truly bad, you have a setting, that takes `True`/`False` and is called boolean. Nobody understands what this means :D

Reply:
For example use the name `check_boolean_operator_problems` or similar